From 3a63b6d9f2df915666a78b7ab80bf1f5861f203e Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Sun, 1 Sep 2024 16:55:47 -0700
Subject: [PATCH] Explicitly set version variable

---
 platforms/libretro/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platforms/libretro/Makefile b/platforms/libretro/Makefile
index 01db018..a5c9298 100644
--- a/platforms/libretro/Makefile
+++ b/platforms/libretro/Makefile
@@ -324,7 +324,7 @@ endif
 
 GIT_VERSION ?= " $(shell git describe --abbrev=7 --dirty --always --tags || echo unknown)"
 ifneq ($(GIT_VERSION)," unknown")
-   CXXFLAGS += -DEMULATOR_BUILD=\"$(GIT_VERSION)\"
+   CXXFLAGS += -DEMULATOR_BUILD=\"1.3.0\"
 endif
 
 include Makefile.common
-- 
2.37.0.windows.1

